packages/core/src/icon/icon.element.scss

```diff
@@ -121,93 +103,11 @@ svg {
   @include flipSVGIcon('vertical');
 }
 
-.clr-i-badge,
-.clr-i-alert {
+.badge,
+.alert {
   fill: var(--badge-color, #{$cds-alias-status-danger});
 }
 
-// outlined
```

Comment:
Now that the svg is conditionally rendered in the template the css is no longer needed to hide/show